tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +275,15 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
     # *********************************************************************
     # ****** Complete Assert Portion of this test **********
     # *********************************************************************
     # Assertions should check:
     # - That result is falsy
     # - That tai and jesse's inventories are the correct length
     # - That all the correct items are in tai and jesse's inventories
+    assert not result
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert set(tai.inventory) == set([item_a, item_b, item_c])
+    assert set(jesse.inventory) == set([item_d, item_e, item_f])
```

Comment:
Same as before, make sure you understand what the lists will look like as a result of a swap rather than casting to a set!